lib/media/streaming_engine.js

```diff
@@ -864,6 +864,11 @@ shaka.media.StreamingEngine = class {
       return;
     }
 
+    const CrossBoundaryStrategy = shaka.config.CrossBoundaryStrategy;
+    if (this.config_.crossBoundaryStrategy !== CrossBoundaryStrategy.KEEP) {
+      this.forwardTimeForCrossBoundary();
+    }
+
```

Comment:
Maybe we could move this a few lines down, after `if (!newTimeIsBuffered(type)) {`. What do you think?

Reply:
Agree, bumped it all the way down.

Reply:
I was referring inside of the if that checks whether the seek is within what we have in the buffer or not... sorry!

Reply:
I prefer not to as that check runs for each mediaState' type separately, which we don't need. Hope that's ok!